modules/microshift-embed-app-rpms-tutorial.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: CONCEPT
+[id="microshift-embed-app-rpms-tutorial_{context}"]
+= Embed application RPMs tutorial
+
+The following tutorial reviews the {product-title} installation steps and adds a description of the workflow for embedding applications. If you are already familiar with `rpm-ostree` systems such as {op-system-ostree-first} and {product-title}, you can go straight to the procedures.
+
+[id="microshift-installation-workflow-review"]
+== Installation workflow review
+The steps used to embed {product-title} into a {op-system-ostree} image can help you understand the steps needed to embed an application. Embedding applications requires a similar workflow.
+
+//larger concept image here
+
+1. To review: To embed {product-title} on {op-system-ostree}, you added the {product-title} repositories to Image Builder. Then, you created a blueprint that declared all the RPMs, container images, files and customizations you needed, including the addition of {product-title}.
+
+2. Then, you added the blueprint to Image Builder and ran a build with the Image Builder CLI tool (`composer-cli`). This step created `rpm-ostree` commits, which were used to create the container image. This image contained {op-system-ostree}.
+
+3. Next, you added the installer blueprint to Image Builder to create an `rpm-ostree` image (ISO) to boot from. This build contained both {op-system-ostree} and {product-title}.
+
+4. Then, you downloaded the ISO with {product-title} embedded, prepared it for use, provisioned it, then installed it onto your edge devices.
+
+[id="microshift-embed-app-rpms-workflow"]
+=== Embed application RPMs workflow
```

Comment:
Can we make this same content level as "Installation workflow review", so that it appears in the Table Of Content please?